### ISSUE TITLE
bug probleme

### ISSUE THREAD

**Issue by demon57730** [ldtteam/minecolonies#1703]
### Minecolonies version

minecolonies-universal-1.11.2-0.8.5090.jar 
and
minecolonies-universal-1.11.2-0.8.5085.jar 

### Expected behavior

impossible to use the construction tool to put any building when I validate it speaks to me as if I wanted to ask the boat while these for the town hall, the builder's building and any other building the tool does not work

**Comment by Raycoms** [comment]
Didn't understand a word you said, can you rephrase that or write in
german? Portuguese?

2017-11-26 23:49 GMT+00:00 demon57730 <notifications@github.com>:

**Comment by demon57730** [comment]
unmöglich, das Konstruktionswerkzeug zu verwenden, um irgendein Gebäude zu setzen. Wenn ich es bestätige, spricht es zu mir, als ob ich das Boot setzen wollte, wenn ich das Rathaus, das Gebäude des Baumeisters und jedes andere Gebäude setzen will, das das Werkzeug nicht funktioniert

Google übersetzen

**Comment by Raycoms** [comment]
I didn't mean you should put it on google translate, I meant you should
write it in an understandable way... Like what exactly is happening, with
commas, interpunctuation and proper grammar.
It works for everyone else perfectly.
Do you have the buildings in your inventory.
Did you notice the GUI switchers on the top. you can use them to switch
between folders.

Am 26. November 2017 um 23:54 schrieb demon57730 <notifications@github.com>:

**Comment by demon57730** [comment]
I will make the video to show you these simpler

**Comment by Raycoms** [comment]
perfect

2017-11-27 0:04 GMT+00:00 demon57730 <notifications@github.com>:

**Comment by demon57730** [comment]
https://youtu.be/QqesC72aQE4

**Comment by Raycoms** [comment]
Now I see it. Will get to work on it.

2017-11-27 0:28 GMT+00:00 demon57730 <notifications@github.com>:

**Comment by demon57730** [comment]
these on both version that I gave

thank you
